### PR TITLE
adds tox support for multiple environment

### PR DESCRIPTION
**Pull request by kjaymiller**
## Adds
- tox

## Changes
python version to python3.8.14+

I want to support python3.10 but considering this is being used with azure my hope was we can keep support in step (or higher) with Oryx supported Python versions

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -29,4 +31,23 @@ addopts = "-ra --cov"
 testpaths = [
     "tests"
 ]
-pythonpath = ['.']
+pythonpath = ['src']
```
Comment (kjaymiller) [inline pyproject.toml:34]: This snuck in but I was running into testing issues since the folder structure changed

Comment (pamelafox) [reply pyproject.toml:34]: Makes sense, yeah

File: pyproject.toml [hunk 2]
```diff
@@ -29,4 +31,23 @@ addopts = "-ra --cov"
...
+
+
+# content of: tox.ini , put in same dir as setup.py
```
Comment (pamelafox) [inline pyproject.toml]: The comment looks like copypasta? (Since this repo has no setup.py)

**Review by pamelafox — APPROVED** [review]
One comment about a comment. I never used tox, but heard good things!

File: pyproject.toml (hunk 2 above)
Comment (kjaymiller) [reply pyproject.toml]: It is and should be removed... removing

**Review by pamelafox — APPROVED** [review]

**Pull request merged**